### PR TITLE
wrote README for data

### PR DESCRIPTION
**Pull request by rjpecoraro**

### CODE REVIEW

File: data/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Data
+
+## Ordered protein structures
+
+The <code> rcsb_pdb_sequence*.csv</code> files are drawn from the RCSB Protein Data Bank (https://www.rcsb.org/). Two files were required because they were too large to be downloaded from the server as a single csv file. These structures contain exactly one protein chain and were solved using either x-ray diffraction, electron microscopy, or electron crystallography. Accessed November 9, 2021.
```
Comment (jiaweih) [inline data/README.md]: You can also embed the link here and below .

Comment (rjpecoraro) [reply data/README.md]: Done!

**Comment by rjpecoraro** [comment]
Can I merge this?

**Comment by jiaweih** [comment]
i think so

**Pull request merged**